pkg/config/config.go

```diff
@@ -1646,3 +1662,17 @@ func (c *CustomAttributeMap) Decode(value string) error {
 	fmt.Println(c)
 	return nil
 }
+
+func (i *IncludeMetricsMap) Decode(value string) error {
+	data := []byte(value)
+	fmt.Println(value)
```

Comment:
If we want to log this information we should add it as debug message.

Reply:
True! Thanks